### ISSUE TITLE
Update to Go 1.20

### ISSUE THREAD

**Issue by j-yarlagadda-ds** [fullstorydev/grpcurl#396]
It's been a while after #300, also with release of Go 1.20 and new CVE fixes, would be nice to have a ticket tracking release for GO v1.20.*

Go Release Notes: 
[Go 1.20](https://tip.golang.org/doc/go1.20)
Go Fixes: 
[Announcement](https://groups.google.com/g/golang-announce/c/V0aBFqaFs_E)
[CVE-2022-41723](https://pkg.go.dev/vuln/GO-2023-1571)
